core/CliMulti.php

```diff
@@ -366,12 +367,12 @@ private function executeNotAsyncHttp($url, Output $output)
                 $url .= '&';
             }
 
-            $url .= 'token_auth=' . $tokenAuth;
+            $requestBody = 'token_auth=' . $tokenAuth;
         }
 
         try {
             $this->logger->debug("Execute HTTP API request: "  . $url);
-            $response = Http::sendHttpRequestBy('curl', $url, $timeout = 0, $userAgent = null, $destinationPath = null, $file = null, $followDepth = 0, $acceptLanguage = false, $this->acceptInvalidSSLCertificate);
+            $response = Http::sendHttpRequestBy('curl', $url, $timeout = 0, $userAgent = null, $destinationPath = null, $file = null, $followDepth = 0, $acceptLanguage = false, $this->acceptInvalidSSLCertificate, false, false, 'POST', null, null, $requestBody, $forcePost = true, [], $forcePost = true);
```

Comment:
@diosmosis I think there's one `forcePost` too much? Maybe the first one isn't needed as it passes `true` for the additional headers?

Reply:
Yes, I removed this locally forgot to push.

Reply:
@tsteur pushed